Marsiple/App/Todo Form/TodoFormView.swift

```diff
@@ -94,13 +85,11 @@ private extension TodoFormView {
     }
     
     func setupTextView() {
-        textViewIsEmpty()
+        placeholder = ""
```

Comment:
Why are we setting the `placeholder` value again? It will always have an initial value per `var placeholder = LocalizationKey.TodoForm.titlePlaceholder.localized()`.

Reply:
Any initial initialisation should be done - initially, within `setupViews`. Regardless, wouldn't `placeholder = ""` terminate the value that set `LocalizationKey.TodoForm.titlePlaceholder.localized()`?

Reply:
Yes it would, but that value isn't important. I could've set it to `""`. There was ```                textView.text = LocalizationKey.TodoForm.titlePlaceholder.localized()
                textView.textColor = .lightGray``` there, but i figured it would be shorter if I would just use it like this `placeholder = ""`.